### ISSUE TITLE
Unable to parse VLC created m3u8 playlist files

### ISSUE THREAD

**Issue by erri120** [jellyfin/jellyfin#4439]
**Describe the bug**
VLC encodes `.m3u8` playlist files, eg when having a song at `Aimer/春はゆく - marie/01 - 春はゆく.flac` it gets encoded like this:
```
#EXTINF:304,Aimer - 春はゆく
Aimer/%E6%98%A5%E3%81%AF%E3%82%86%E3%81%8F%20-%20marie/01%20-%20%E6%98%A5%E3%81%AF%E3%82%86%E3%81%8F.flac
```
Jellyfin will read the playlist and is unable to find the file:
```
[2020-11-07 19:04:50.726 +01:00] [WRN] [12] MediaBrowser.Controller.Entities.BaseItem: Unable to find linked item at path "Aimer/%E6%98%A5%E3%81%AF%E3%82%86%E3%81%8F%20-%20marie/01%20-%20%E6%98%A5%E3%81%AF%E3%82%86%E3%81%8F.flac"
```

**System (please complete the following information):**
 - OS: Windows
 - Jellyfin Version: 10.6.4
 - Storage: local

**To Reproduce**
1. Create a VLC `m3u8` playlist file
2. Let Jellyfin try and parse it

**Expected behavior**
Parse these files correctly.

**Logs**
```
[2020-11-07 19:04:50.726 +01:00] [WRN] [12] MediaBrowser.Controller.Entities.BaseItem: Unable to find linked item at path "Aimer/%E6%98%A5%E3%81%AF%E3%82%86%E3%81%8F%20-%20marie/01%20-%20%E6%98%A5%E3%81%AF%E3%82%86%E3%81%8F.flac"
```

**Comment by dkanada** [comment]
An issue here will probably get more visibility, and it wouldn't be the first time we managed to find someone to help with updates or features on a random library :) let's see what happens if this stays open.

**Comment by erri120** [comment]
I mean, I could prob just update the lib myself it's just that this is a rather niche problem and I wanna know from the author if this is even worth adding.

**Comment by erri120** [comment]
PR got merged and new version is available: [NuGet](https://www.nuget.org/packages/PlaylistsNET/1.1.3) issue can be closed when deps are updated.

**Comment by erri120** [comment]
Closed with 809556eb98d3d57ac2e5b4c50ccdccdc70389e07
